### ISSUE TITLE
Sprite3D and Quad in one Viewport increases CPU usage

### ISSUE THREAD

**Issue by Sipaha** [okamstudio/godot#1262]
When I create empty project and add at scene one of Sprite3D or Quad it works great and CPU usage remains normal (~2-3%):

![good_proc](https://cloud.githubusercontent.com/assets/3995047/5792403/563442d2-9f49-11e4-95a5-b8ded3a27e82.png)

But when I add both of them at scene, CPU usage increases to ~26%:

![bad_proc](https://cloud.githubusercontent.com/assets/3995047/5792411/88e2a82c-9f49-11e4-823b-81019bfdf05a.png)

My platform is Windows.

**Comment by akien-mga** [comment]
Can you still reproduce this issue in Godot 1.1 or the current master branch?

**Comment by Sipaha** [comment]
Yes, problem still exist in Godot_v2.0_alpha_20150930_win64.exe. I create Quad, Sprite3D with texture and turn on Update Always in editor to reproduse the problem.

**Comment by vnen** [comment]
Tried this and there's a massive flood of errors in the console (couldn't copy because Windows console is awful).

When I close, the editor keeps flooding this errors:

```
ERROR: RasterizerGLES2::set_render_target:  Condition ' rt->fbo==0 ' is true.
At:  drivers\gles2\rasterizer_gles2.cpp:4375
ERROR: VisualServerRaster::_draw_viewport:  Condition ' desired_rect.width<=0 ||
 desired_rect.height<=0 ' is true.
At:  servers\visual\visual_server_raster.cpp:7003
````

Sample project: https://dl.dropboxusercontent.com/u/9300401/1262.zip

**Comment by akien-mga** [comment]
Not critical for the upcoming 2.1, so moving to the next milestone.

**Comment by akien-mga** [comment]
What's the status on this in the current master branch?
